modestpy/estim/ga/ga.py

```diff
@@ -90,6 +82,11 @@ def __init__(self, fmu_path, inp, known, est, ideal,
                          Lating Hypercube Sampling.
         """
         self.logger = logging.getLogger(type(self).__name__)
+
+        deprecated_msg = 'This GA implementation is deprecated. Use MODESTGA instead.'
```

Comment:
These warnings show up in validation, when the validation period does not start from 0s. Not sure how to eliminate them yet.

Reply:
These warnings are not printed under Linux (WSL2, Ubuntu 18.04).

Reply:
I submited an issue on FMPy: https://github.com/CATIA-Systems/FMPy/issues/179

Reply:
So the warnings come from the Dymola's FMU and SDU should contact Dymola if you want to get a patch: https://github.com/CATIA-Systems/FMPy/issues/179
Otherwise, it will be probably fixed in the next Dymola release.

It does not seem these warnings affect the simulation results, so we can leave it as it is.